packages/components/src/core/Button/index.stories.tsx

```diff
@@ -193,7 +210,7 @@ export const LivePreview = {
                 type={type}
                 icon={icon}
                 disabled={disabled}
-                key={disabled}
+                key={String(disabled)}
```

Comment:
key has to be either a `number` or `string`, so I use `String()` to convert the value to `string`!

https://www.w3schools.com/jsref/jsref_string.asp

Reply:
Oh, perfect, thanks!!